File: LYLAcademy.API/LYLAcademy.API/Models/Calendar.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace LYLAcademy.API.Models
{
    public class Calendar
    {
        public Calendar()
        {
            Course = new Course();
            Teacher = new Teacher();
        }
        public int CalendarId { get; set; }
        public int CourseId { get; set; }
        public int TeacherId { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }     
        public string Year { get; set; }     
        public string CourseContent { get; set; }     
        public decimal Price { get; set; }     
        public int Status { get; set; }     
        public int IsDelete { get; set; }     
        public DateTime CreateDate { get; set; }     
        public DateTime UpdateDate { get; set; }
        public Course Course { get; set; }
        public Teacher Teacher { get; set; }

    }
}

File: LYLAcademy-SPA/src/app/models/Calendar.ts
import { Course } from './Course';
import { Participant } from './Participant';
import { Teacher } from './Teacher';

export class Calendar {
    calendarId:number;
    courseId:number;
    teacherId:number;
    year:string;
    courseContent:string;
    price:number;
    status:number;
    createDate: Date;
    updateDate: Date;
    startDate: Date;
    endDateStr: string;
    startDateStr: string;
    endDate: Date;
    course: Course;
    teacher:Teacher;   
    participantList:Participant[];
    isDelete:number;
    participantCount:number;
}
